sftp-server.c

```diff
@@ -51,14 +51,18 @@
 
 #include "sftp.h"
 #include "sftp-common.h"
-
+#ifdef PRIVSEP_AUTH_CHILD_LOG_NOT_SUPPORTED
+#include "atomicio.h"
+#endif
 char *sftp_realpath(const char *, char *); /* sftp-realpath.c */
 
 /* Maximum data read that we are willing to accept */
 #define SFTP_MAX_READ_LENGTH (SFTP_MAX_MSG_LENGTH - 1024)
 
 /* Our verbosity */
 static LogLevel log_level = SYSLOG_LEVEL_ERROR;
+static SyslogFacility log_facility = SYSLOG_FACILITY_AUTH;
```

Comment:
encapsulate inside windows block

Reply:
instead of modifying the existing code that's common to both windows and linux.
Add new variables like log_facility_g, log_stderr_g here. 
Assign them at line 1784.